datumaro/plugins/open_images_format.py

```diff
@@ -238,6 +289,53 @@ def _load_labels(self, items_by_id):
                     item.annotations.append(Label(
                         label=label_index, attributes={'score': confidence}))
 
+    def _load_bboxes(self, items_by_id):
+        label_categories = self._categories[AnnotationType.label]
+
+        for bbox_path in self._glob_annotations(
+            '*' + OpenImagesPath.BBOX_DESCRIPTION_FILE_SUFFIX
+        ):
+            with self._open_csv_annotation(bbox_path) as bbox_reader:
+                for bbox_description in bbox_reader:
+                    image_id = bbox_description['ImageID']
+                    item = items_by_id[image_id]
+
+                    label_name = bbox_description['LabelName']
+                    label_index, _ = label_categories.find(label_name)
+                    if label_index is None:
+                        raise UndefinedLabel(
+                            item_id=item.id, subset=item.subset,
+                            label_name=label_name, severity=Severity.error)
```

Comment:
I think, the severity argument should be separate from the error class. Probably, I see reporting a dataset problem as an `Extractor` (`Converter`) - base class - method, which could have such severity parameter. Then, it could be handled according to the current error control policy - e.g. a warning could be promoted to an error or ignored. #142

Reply:
I don't understand - do you want me to change something in the code? I didn't implement the `UndefinedLabel` class, I'm just using it.

Reply:
Probably, this error class is not supposed to be used outside dataset validators. But we can leave it.